Automattic/wp-calypso
# Sites Dashboard: Site thumbnail component

#### Proposed Changes

* Add component to generate site thumbnails for public, private and coming soon sites.
* Component is only apart of the dev docs and is not used any where at the moment

#### Testing Instructions

- Checkout branch and browse `http://calypso.localhost:3000/devdocs/design` 
- Search for the SiteThumbnail component
- Play around with the previews in `packages/components/src/site-thumbnail/docs/example.jsx` changing background color and checking the text color changes to be visible on dark or light background.
- toggle network speed in chrome network tabs to check the loader is shown.



![image](https://user-images.githubusercontent.com/47489215/180333173-9c3b0ecb-c362-4a4c-9256-7208c6b1ef7c.png)


#### Pre-merge Checklist






Related to https://github.com/Automattic/wp-calypso/issues/65172
